### ISSUE TITLE
KeyError: 'https://json.schemastore.org/base.json'

### ISSUE THREAD

**Issue by imphil** [henryiii/validate-pyproject-schema-store#9]
I'm getting the following error when I attempt to run pre-commit:

```
❯ pre-commit run -a
[INFO] Initializing environment for https://github.com/abravalheri/validate-pyproject.
[INFO] Initializing environment for https://github.com/abravalheri/validate-pyproject:validate-pyproject[all],validate-pyproject-schema-store==2024.1.29.
[INFO] Installing environment for https://github.com/abravalheri/validate-pyproject.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
ruff.....................................................................Passed
ruff-format..............................................................Passed
clang-format.............................................................Passed
trim trailing whitespace.................................................Passed
mixed line ending........................................................Passed
fix end of files.........................................................Passed
Validate pyproject.toml..................................................Failed
- hook id: validate-pyproject
- exit code: 1

[ERROR] KeyError: 'https://json.schemastore.org/base.json'
```

I haven't been able to dig much deeper, maybe you see something wrong right away.

https://github.com/cocotb/cocotb/pull/3618 has the code showing this problem, and https://github.com/imphil/cocotb/blob/validate-pyproject/pyproject.toml is the associated pyproject.toml (just run `pre-commit run -a` in this branch and you should see the issue).

**Comment by henryiii** [comment]
Ahh, Poetry was updated to reference a license from `base.json`, which we don't include. Will need to figure out what we should do with that. The previous version should work until we fix it.

**Comment by henryiii** [comment]
I’ve yanked the current release. I’ll patch it for the b next release, and work on fixing it properly in validate-pyproject.

**Comment by henryiii** [comment]
Fixed in `2024.01.31`. Stripping it for now. I also added a test of validate-pyproject (when this was started, 0.16 wasn't released yet), so we should be able to detect when major breakages happen in SchemaStore and fix it before merging the update.
